hcloud/actions/client.py

```diff
@@ -37,6 +42,8 @@ class ActionsPageResult(NamedTuple):
 
 
 class ActionsClient(ClientEntityBase):
+    _client: Client
```

Comment:
Just to confirm, this works because this is a [Type annotation for local variable](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING) and is therefore not evaluated by the runtime?

Reply:
Yes.

We could also write:
```py
class ActionsClient(ClientEntityBase):
    _client: "Client"
```

And not import `from __future__ import annotations`.

The TYPE_CHECKING thingy is mostly here to prevent circular imports and large runtime imports.